fullcontact/schema/person_schema.py

```diff
@@ -49,9 +56,8 @@ class ProfileSchema(BaseCombinationSchema):
     )
 
 
-class PersonSchema(BaseSchema):
-    schema_name = "Person"
-
+class PersonSummarySchema(BaseSchema):
+    schema_name = "Person Summary"
     email: str
     emails: List[str]
     phone: str
```

Comment:
We can technically take in multiple phones as input
And given email or emails, is it worth collapsing email and emails to just a list?

Reply:
Then again email is a nice shortcut for a singleton list for emails. so maybe that's okay.

Reply:
Just giving an option! Even if the string parameter is removed, the users can pass in a list with a single element. I'll remove it. It does look redundant.

Reply:
Or, on second thought, I'll leave it there. It looks harmless and useful to some!